### ISSUE TITLE
Requires high perf GPU

### ISSUE THREAD

**Issue by specialunderwear** [meetfranz/plugins#217]
Why is the Requires high perf GPU flag YES for franz? Because of this, when franz is active osx can not swap to the low energy GPU, causing the battery to be unnecessarily taxed.

**Comment by cynicer** [comment]
This is still open? I think this is quite serious. Until its fixed I won't recommend Franz to any Mac user, because it reduces the battery life easily by 1-2 hours.

**Comment by specialunderwear** [comment]
See for yourself in process monitor

**Comment by joltcan** [comment]
Still an issue at this date :(

**Comment by timrettop** [comment]
Yes still an issue.  You can kill the helper that uses the discrete graphics after it starts, I'm not sure which services it would affect, but the ones I use, slack, Skype, hangouts and groupme, all seem to still work.

**Comment by rfrowe** [comment]
This is still an issue. I love Franz but this forces me not to use it. I usually leave Franz open all the time, but this means my laptop is _always_ using the high performance GPU which drains the battery much faster.

There's no reason a glorified chrome wrapper should need my AMD Radeon R9 M370X. Why has this issue not been addressed by any Franz developers? I also filed a ticket through zendesk over a week ago with no response.
